common/static/js_test.yml

```diff
@@ -42,13 +42,13 @@ lib_paths:
     - js/test/add_ajax_prefix.js
     - js/test/i18n.js
     - coffee/src/jquery.immediateDescendents.js
+    - js/vendor/require.js
```

Comment:
Should add a comment explaining why this is necessary (it's confusing because these are supposed to be tests of things not using requirejs).